Add an informal privacy policy, some help on errors and some location docs.

obra opened the pull request:
A friend mentioned on twitter that they don't trust aqi.wtf because it's asking for location information and seems sketchy. 

I figured it was time for an explicit privacy policy. PLEASE MAKE SURE THIS PRIVACY POLICY REFLECTS YOUR INTENT AND THE FACTS ON THE GROUND.

Also, I tweaked some messages and added a better fallback if the user says no to location tracking.

obra committed:
Add an informal privacy policy to try to address some fearmongering on the twitters

obra committed:
Make it possible to pass in an explanation to a state message.

obra committed:
Switch to a new 'announceError method, which gives us some rope for later. Also update the unsupported message to point at PA map. Most contentiously, change the 'can't locate you' message to an 80s kids tv reference

obra committed:
Add a note about why we're stalking you to the "finding you" screen

skalnik commented:
Nice, this is great! I've also wanted to add some details about why location data is requested and all of that, thanks. I'll dig into this more in depth tonight and try to get it merged.

obra commented:
Great. Feel free to just hack it to death and merge the version that matches your intent.

skalnik committed:
Merge branch 'main' of https://github.com/skalnik/aqi-wtf into why-we-need-loc

skalnik committed:
Reformat and style privacy page 🍝

skalnik committed:
We don't need to interpolate in these strings 🧵

skalnik committed:
Shorten name a bit for space 👽

skalnik committed:
Small tweak to goals 🥅

skalnik committed:
Consistently use `PurpleAir` 💜

Appears this is their preferred spelling, so lets use that!

skalnik committed:
Update metadata for page 💽

1. I hate that this is a thing that exists in 2020
2. Unsure why people would share this, but might as well be prepared 🤷‍♂️

skalnik committed:
Doesn't need to cover either 📔

skalnik committed:
Wait this should've been removed 🙈

skalnik commented:
Thanks again for this! I love the new Scooby-Doo message 🐕
